firebase.json

```diff
@@ -0,0 +1,16 @@
+{
+  "hosting": {
+    "public": "build",
```

Comment:
So the [public](https://firebase.google.com/docs/hosting/full-config#public) attribute specifies which directory to deploy. Is "build" the directory created by React?

Reply:
Nvm I found [this doc](https://create-react-app.dev/docs/deployment/) that explains it